src/operator/scan.ts

```diff
@@ -32,23 +32,23 @@ import {Subscriber} from '../Subscriber';
  * @see {@link mergeScan}
  * @see {@link reduce}
  *
- * @param {function(acc: R, value: T): R} accumulator The accumulator function
+ * @param {function(acc: R, value: T, , index: number): R} accumulator The accumulator function
```

Comment:
@kwonoj Can you fix these commas? Should be `, ` not `, , `. Also limiting the line width to 80 chars would be good.